clusterloader2/testing/list/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+{{$image := DefaultParam .CL2_BENCHMARK_IMAGE "gcr.io/k8s-testimages/perf-tests-util/request-benchmark:latest"}}
```

Comment:
Where is the source code for this image? Would help to link that in a comment here so folks can cross-reference.

Reply:
Will add.
It's here: https://github.com/kubernetes/perf-tests/tree/master/util-images/request-benchmark